<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPoints</class>
 <widget class="QDialog" name="ExportPoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Points</string>
  </property>
  <widget class="QPushButton" name="ExportB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelB">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileDestinationLE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PointFileDestinationL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point File Destination:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectPointGroupL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Point Groups:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EastingL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Easting:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DescriptionLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DelimiterCB">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>100</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Comma</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="PointNameLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="ElevationL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PFFL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>62</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Specify Point File Format:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EastingLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="NorthingL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Northing:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NorthingLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DelimiterL">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delimiter:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PointNameL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ElevationLE">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseB">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>30</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QListWidget" name="PointGroupsLW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
